src/storage/mod.rs

```diff
@@ -2614,22 +2628,13 @@ impl<E: Engine, L: LockManager, F: KvFormat> Storage<E, L, F> {
         }
         let provider = self.causal_ts_provider.clone();
         let sched = self.get_scheduler();
-        let concurrency_manager = self.get_concurrency_manager();
         self.sched_raw_command(CMD, async move {
             // Raw atomic cmd has two locks, one is concurrency_manager and the other is txn
```

Comment:
These comments may not be in the correct place anymore.

Reply:
Yesh, these comments should be moved, thanks!